packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -771,9 +767,7 @@ export class AttributesController extends ConsumptionBaseController {
         }
 
         if (typeof successor.succeeds !== "undefined" && !predecessorId.equals(successor.succeeds.toString())) {
-            return ValidationResult.error(
-                CoreErrors.attributes.invalidPredecessor("The predecessor's id and the explicitly set value for the successor's succeeds field don't match.")
-            );
+            return ValidationResult.error(CoreErrors.attributes.differentPredecessorNamedInSuccessor());
```

Comment:
I find the term "name" ambiguous here. Maybe better `setPredecessor(Id)DoesNotMatchActualPredecessor(Id)` or similar.